docs/BestPractices.md

```diff
@@ -3,6 +3,8 @@ title: "Best Practices"
 layout: page
 ---
 
+<link href="/assets/css/syntax.css" rel="stylesheet" >
```

Comment:
What is different or better in this syntax highlighting over the existing highlighting in this project?